tests/python/frontend/pytorch/test_forward.py

```diff
@@ -0,0 +1,851 @@
+    verify_model('resnet34')
+
+def test_resnet50():
+    torch.set_grad_enabled(False)
+    verify_model('resnet50')
+
+def test_resnet101():
+    torch.set_grad_enabled(False)
+    verify_model('resnet101')
+
+def test_resnet152():
+    torch.set_grad_enabled(False)
+    verify_model('resnet152')
+
+def test_squeezenet1_0():
+    torch.set_grad_enabled(False)
+    verify_model('squeezenet1_0')
+
+def test_squeezenet1_1():
+    torch.set_grad_enabled(False)
+    verify_model('squeezenet1_1')
+
+def test_mobilenet_v2():
+    torch.set_grad_enabled(False)
+    verify_model('mobilenet_v2')
+
+def test_densenet121():
+    torch.set_grad_enabled(False)
+    verify_model('densenet121')
+
+def test_densenet161():
+    torch.set_grad_enabled(False)
+    verify_model('densenet161')
+
+def test_densenet169():
+    torch.set_grad_enabled(False)
+    verify_model('densenet169')
+
+def test_densenet201():
+    torch.set_grad_enabled(False)
+    verify_model('densenet201')
+
+def test_inception_v3():
+    torch.set_grad_enabled(False)
+    verify_model('inception_v3')
+
+def test_googlenet():
+    torch.set_grad_enabled(False)
+    verify_model('googlenet')
+
+def test_mnasnet0_5():
+    torch.set_grad_enabled(False)
+    verify_model('mnasnet0_5')
+
+def test_mnasnet1_0():
+    torch.set_grad_enabled(False)
+    verify_model('mnasnet1_0')
+
+#TODO: Fix VGG and AlexNet issues (probably due to pooling)
+"""
+def test_alexnet():
+    torch.set_grad_enabled(False)
+    verify_model('alexnet')
+
+def test_vgg11():
+    torch.set_grad_enabled(False)
+    verify_model('vgg11')
+
+def test_vgg13():
+    torch.set_grad_enabled(False)
+    verify_model('vgg13')
+
+def test_vgg16():
+    torch.set_grad_enabled(False)
+    verify_model('vgg16')
+
+def test_vgg19():
+    torch.set_grad_enabled(False)
+    verify_model('vgg19')
+
+def test_vgg11_bn():
+    torch.set_grad_enabled(False)
+    verify_model('vgg11_bn')
+
+def test_vgg13_bn():
+    torch.set_grad_enabled(False)
+    verify_model('vgg13_bn')
+
+def test_vgg19_bn():
+    torch.set_grad_enabled(False)
+    verify_model('vgg19_bn')
+"""
+
+if __name__ == '__main__':
+
+    # Single operator tests
+    test_forward_add()
+    test_forward_subtract()
+    test_forward_multiply()
+    test_forward_unsqueeze()
+    test_forward_concatenate()
+    test_forward_relu()
+    test_forward_adaptiveavgpool()
+    test_forward_maxpool()
+    test_forward_hardtanh()
+    test_forward_conv()
+    test_forward_threshold()
+    test_forward_contiguous()
+    test_forward_batchnorm()
+    test_forward_transpose()
+    test_forward_size()
+    test_forward_view()
+    test_forward_select()
+    test_forward_clone()
+    test_forward_logsoftmax()
+    test_forward_sigmoid()
+    test_forward_dense()
+    test_forward_avgpool()
+    test_forward_dropout()
+    test_forward_slice()
+    test_forward_mean()
+    test_forward_expand()
+    test_forward_pow()
+    test_forward_chunk()
+
+    # Model tests
+    test_resnet18()
```

Comment:
I don't have strong preference, but I think we can probably just have a test_resnet and have different layers this function.

Having as many as possible tests it great. One (minor) concern I have is if it is necessary to have tests for all these variants (i.e. ResNet 101 and ResNet 152) because it will significantly slow down the CI (although I again agree it would be a different story for nightly tests). @tqchen  @icemelon9 @masahi How do you guys think?

Reply:
yeah I agree, having one variant per different network is enough @alexwong can you remove or comment out `test_resnet34, 50, test_densenet_161` etc?

Reply:
Will do.